lib/CGameInfoCallback.cpp

```diff
@@ -479,6 +479,17 @@ std::vector <const CGObjectInstance *> CGameInfoCallback::getVisitableObjs(int3
 
 	return ret;
 }
+
+std::vector<ConstTransitivePtr<CGObjectInstance>> CGameInfoCallback::getAllVisitableObjs() const
+{
+	std::vector<ConstTransitivePtr<CGObjectInstance>> ret;
+	for(auto & obj : gs->map->objects)
+		if(obj->isVisitable() && obj->ID != Obj::EVENT && getTile(obj->pos, false))
```

Comment:
`getTile` is needed to filter visible objects? There should be `isVisible(int3)` and `isVisible(CGObjectInstance*)` methods specifically for this.
```suggestion
		if(obj->isVisitable() && obj->ID != Obj::EVENT && isVisible(obj))
```

Reply:
Yes, for visible objects. Thanks. That's a better fitting function. :)